libraries/Wire/Wire.h

```diff
@@ -28,9 +28,7 @@
 
 #include "RingBuffer.h"
 
-#define BUFFER_LENGTH 32
```

Comment:
This is a left over from the AVR Wire implementation and is also incorrect for this core, as the size of the ring buffers used is 64.